wordpress-mobile/WordPress-iOS
# Lock Screen Widgets: This Week Chart Widget

Fixes #21501

- Created `LockScreenWidgetChartView` which shows a bar chart
- Created Providers and Configs

<img width="380" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/fb0f659d-4c5b-40b2-bbd2-1e161a3cb41c">

Relevant discussion: pc8eDl-14z-p2#comment-935

## To test:

2. Open Jetpack
3. Log into account
4. Go to lock screen, long press, customize, tap on widget area, select Jetpack app
5. Confirm "This Week Views" widget exists
6. Confirm it shows a single field widget for a site with no views data
7. Confirm it shows a 7-day line-area chart and it correctly corresponds to days
8. Confirm a total is shown correctly

## Regression Notes
1. Potential unintended areas of impact

Shouldn't affect other widgets

2. What I did to test those areas of impact (or what existing automated tests I relied on)

None

3. What automated tests I added (or what prevented me from doing so)

None

PR submission checklist:


UI Changes testing checklist:
